### PR TITLE
Added ocp_quickstart_guide for OCP users

### PR DESCRIPTION
**Pull request by chaitanya1731**
Added operator installation steps for RedHat OpenShift Container Platform

Signed-off-by: chaitanya1731 <chaitanya.kulkarni@intel.com>

### CODE REVIEW

**Review by bart0sh — APPROVED** [review]
/lgtm

File: cmd/operator/ocp_quickstart_guide/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+- Third Generation Intel® Xeon® Scalable Platform, code-named “Ice Lake” or later
+- Configure BIOS using below details
+    ![SGX Server BIOS](images/SGX-BIOS.PNG)
+    [**Note:** The BIOS configuration shown above is just for the reference. Please contact your BIOS vendor for details]
+
+## Installation
+### Prerequisites
+- Make sure Red Hat OpenShift Cluster is ready to use and the developing machine is RHEL and `oc` command is installed and configured properly. Please note that the following operation is verified on Red Hat OpenShift Cluster 4.11 and working machine RHEL-8.6
+- Install the `oc` command to your development machine
+- Follow the [link](https://docs.openshift.com/container-platform/4.10/hardware_enablement/psap-node-feature-discovery-operator.html) to install **NFD operator** (if it's not already installed).
```
Comment (mythi) [inline cmd/operator/ocp_quickstart_guide/README.md:28]: `4.10` or `4.11`?

```suggestion
- Follow the [link](https://docs.openshift.com/container-platform/4.11/hardware_enablement/psap-node-feature-discovery-operator.html) to install **NFD operator** (if it's not already installed).   
```

File: cmd/operator/ocp_quickstart_guide/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+### Verify Operator installation
+1.  Go to **Operator** -> **Installed Operators**
+2.  Verify the status of operator as **Succeeded**
+3.  Click **Intel Device Plugins Operator** to view the details  
+    ![Verify Operator](images/verify-operator.PNG)
+
+
+## Deploying Plugins
+The [`v0.24.0`](https://github.com/intel/intel-device-plugins-for-kubernetes/releases/tag/v0.24.0) release of the operator supports only the Intel SGX device plugin. GPU, Intel QAT, Intel DSA, Intel  IAA, and other device plugins will be supported in future releases.
```
Comment (mythi) [inline cmd/operator/ocp_quickstart_guide/README.md:52]: this repeats what was written in the intro so maybe remove?

File: cmd/operator/ocp_quickstart_guide/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+### Intel SGX Device Plugin
+
+**Prerequisites**
```
Comment (mythi) [inline cmd/operator/ocp_quickstart_guide/README.md:56]: this was already part of the pre-requisites above so can be dropped

File: cmd/operator/ocp_quickstart_guide/README.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+                SGX: {op: Exists}
+                SGXLC: {op: Exists}
+            - feature: cpu.sgx
+                matchExpressions:
+                enabled: {op: IsTrue}
+            - feature: kernel.config
+                matchExpressions:
+                X86_SGX: {op: Exists}
+    ```
+- **Deploying SELinux Policy**
```
Comment (mythi) [inline cmd/operator/ocp_quickstart_guide/README.md:91]: move to general pre-requisites

File: cmd/operator/ocp_quickstart_guide/README.md (hunk 1 above)
Comment (chaitanya1731) [reply cmd/operator/ocp_quickstart_guide/README.md]: Done

File: cmd/operator/ocp_quickstart_guide/README.md (hunk 2 above)
Comment (chaitanya1731) [reply cmd/operator/ocp_quickstart_guide/README.md]: Done

File: cmd/operator/ocp_quickstart_guide/README.md (hunk 3 above)
Comment (chaitanya1731) [reply cmd/operator/ocp_quickstart_guide/README.md]: Done

File: cmd/operator/ocp_quickstart_guide/README.md (hunk 4 above)
Comment (chaitanya1731) [reply cmd/operator/ocp_quickstart_guide/README.md]: Done

File: cmd/operator/ocp_quickstart_guide/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+## Minimum Hardware Requirements
+### Intel SGX Enabled Server
+- Third Generation Intel® Xeon® Scalable Platform, code-named “Ice Lake” or later
+- Configure BIOS using below details
+    ![SGX Server BIOS](images/SGX-BIOS.PNG)
+    [**Note:** The BIOS configuration shown above is just for the reference. Please contact your BIOS vendor for details]
+
+## Installation
+### Prerequisites
+- Make sure Red Hat OpenShift Cluster is ready to use and the developing machine is RHEL and `oc` command is installed and configured properly. Please note that the following operation is verified on Red Hat OpenShift Cluster 4.11 and working machine RHEL-8.6
```
Comment (mythi) [inline cmd/operator/ocp_quickstart_guide/README.md:26]: How long we want to mention 4.10?

```suggestion
- Make sure Red Hat OpenShift Cluster is ready to use and the developing machine is RHEL and `oc` command is installed and configured properly. Please note that the following operation is verified on Red Hat OpenShift Cluster 4.11 and working machine RHEL-8.6. Version 4.10 is also known to work but requires additional SELinux configurations to work (see below).
```

**Review by mythi — COMMENTED** [review]
This looks good to me but one question about 4.10 vs 4.11.

**Review by mythi — APPROVED** [review]

**Pull request merged**